pkg/render/manager.go

```diff
@@ -431,7 +433,7 @@ func (c *managerComponent) managerProxyContainer() corev1.Container {
 		{Name: "VOLTRON_ENABLE_MULTI_CLUSTER_MANAGEMENT", Value: strconv.FormatBool(c.cfg.ManagementCluster != nil)},
 		{Name: "VOLTRON_TUNNEL_PORT", Value: defaultTunnelVoltronPort},
 		{Name: "VOLTRON_DEFAULT_FORWARD_SERVER", Value: "tigera-secure-es-gateway-http.tigera-elasticsearch.svc:9200"},
-		{Name: "VOLTRON_ENABLE_COMPLIANCE", Value: fmt.Sprintf("%v", c.cfg.ComplianceFeatureActive)},
+		{Name: "VOLTRON_ENABLE_COMPLIANCE", Value: strconv.FormatBool(c.cfg.Compliance != nil && c.cfg.ComplianceFeatureActive)},
```

Comment:
We are letting manager handle the state of the feature (Compliance) based on the license, but why does the operator need to handle the state of the feature for voltron? Seems like we should do the same as the manager UI and count on the component understanding the license and if the feature is enabled or not.

What is enabled/disabled with this env var?

Reply:
This env var is used [here](https://github.com/tigera/calico-private/blob/e2b4d67f589f0889baacd956e7c6b789d23d26d8/voltron/cmd/voltron/main.go#L189) - it registers compliance as a target for the default proxy (i.e. proxy within management cluster IIUC).

It appears to be the pattern for Voltron to be passed feature enable decisions via env var rather than make those decision itself